### DISCUSSION TITLE
Tell me your favorite video games🎮あなたの好きなゲームを教えてください

### DISCUSSION THREAD

**Discussion by ishizuka-yuk** [NAIST-SE/AttractingContributors#500] [General]
Let us know what games you like to play.
It doesn't matter what genre, so feel free to send us anything!

あなたの好きなゲームを教えてください。
ジャンルは問わないので何でも気軽に送ってください！！

Here's my repo : https://github.com/yukidaruma4869/FavoriteVideoGames.git

**Comment by tmymzkr** [comment]
https://github.com/m-tomoya/AttractingContributors-movie
